config/ipam/image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: quay.io/metal3-io/ip-address-manager:v0.1.1
+      - image: quay.io/metal3-io/ip-address-manager:latest
```

Comment:
what is that for?

Reply:
isn't it better to pin the version?

Reply:
we do once we have image in quay with release version tag pinned, for now we do not have release with v1beta1 API changes so we use latest(once we release ipam v1.0.X from main branch, we should have quay image:  quay.io/metal3-io/ip-address-manager:v1.0.0 which we will reference from here)

Reply:
I see. Makes sense
/lgtm